src/test/java/uk/nhs/prm/e2etests/test/RepositoryE2ETest.java

```diff
@@ -1,10 +1,7 @@
 package uk.nhs.prm.e2etests.test;
 
 import lombok.extern.log4j.Log4j2;
-import org.junit.jupiter.api.BeforeAll;
-import org.junit.jupiter.api.Disabled;
-import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.*;
```

Comment:
Can we opt for single line imports over wildcard please!

Reply:
PR addressed! Thanks